File: python/1-50/1.py


def multOf3and5(ceiling):
    total = 0
    result = sum(x for x in range(ceiling) if x % 5 == 0 or x % 3 == 0)
    for x in range(ceiling):
        if x % 5 == 0 or x % 3 == 0:
            total += x

    print(total)


multOf3and5(1000)

File: javascript/1-50/1.js
function sumMultiplesOfThreeAndFive(ceiling) {
  var result = 0;

  for (var i = 0; i < ceiling; i++){
    if (i % 3 == 0 || i % 5 == 0 ){
      result += i;
    }}
    console.log(result)
  }


sumMultiplesOfThreeAndFive(1000);
